homeworks/08-functional-reactive-programming/src/components/Grid.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './Grid.css';
+
+import { Cat } from './Cat';
+import { Wall } from './Wall';
+import { Window } from './Window';
+
+import { Cell, CellContent } from '../types';
+
+interface GridProps {
+  cells: Cell[],
+  onClick: (content: CellContent) => void,
+}
+
+export const Grid: React.FC<GridProps> = ({ cells, onClick }) => {
+  // eslint-disable-next-line no-undef
+  const gridCells: JSX.Element[] = [];
+  cells.forEach((cell) => {
```

Comment:
Please use map instead of `forEach` and `push`.

Reply:
Done in https://github.com/vbarbasins/evo-ts-bootcamp/pull/9/commits/be64d87757c001a17c7282b5e6d16d25a37c48b9